check-peer-deps.js

```diff
@@ -167,8 +169,29 @@ const checkAllPeerDeps = async () => {
   return Promise.all(promises);
 };
 
+const setOpts = () => {
+  const opts = commandLineArgs([
+    { name: 'debug', alias: 'd', type: Boolean },
+    { name: 'no-include-dev', type: Boolean },
+    { name: 'max-retries', type: Number },
+    { name: 'help', alias: 'h', type: Boolean, defaultValue: false }
+  ]);
+
+  DEBUG = opts['debug'] ? opts['debug'] : false;
```

Comment:
As stated above, let's use the `defaultValue` property of the `option-descriptions` to determine the value here... assuming it doesn't automatically set it for us (which I imagine it will).